about.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="assets/css/styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Outfit:wght@100;400;700&family=Poppins&display=swap" rel="stylesheet">
+  <title>About</title>
+  <script src="assets/js/about.js" defer></script>
+</head>
+<body>
+  <header>
+        <!--DOM mobile menu-->
+        <div class="mobile-menu">
+
+        </div>
+    <nav class="nav-desktop">
+      <ul class="nav-black">
+        <li><i class="fab fa-facebook-f"></i></li>
+        <li><i class="fab fa-twitter"></i></li>
+        <li><span>English</span></li>
+        <li><span>My Page</span></li>
+        <li><span>Logout</span></li>
+      </ul>
+      <div class="nav-grey">
+        <a href="./index.html">
+          <img src="assets/images/ww.png">
+        </a>
+        <ul> 
+          <li><a class="link" href="./about.html">About</a></li>
```

Comment:
- [ ] [OPTIONAL] Since the user is already on the about page, please change the text and `href` to link back to the home page.